index.html

```diff
@@ -14,19 +14,36 @@
 </head>
 
 <body>
+
     <header>
         <a href="#">Kemboi</a>
         <nav>
             <img class="header-icon" src="images/union.svg" />
             <ul>
-                <li><a href="#">Portfolio</a></li>
-                <li><a href="#">About</a></li>
-                <li><a href="#">Contact</a></li>
+                <li><a href="#portfolio">Portfolio</a></li>
+                <li><a href="#about">About</a></li>
+                <li><a href="#contact">Contact</a></li>
                 <img class="header-mail" src="images/mail.svg" alt="Email" />
             </ul>
         </nav>
     </header>
 
+    <div class="navMenu" id="navMenu">
+        <button class="menuBtn">
+            <img src="./images/cross.svg" alt="">
+        </button>
```

Comment:
👏 Great job on adding the close icon, it's functional and working well! However, there are some style adjustments needed to match the design requirements:

- [ ] Please add proper styling for the close Button. Try to adjust spacing, layout, colors, background of the close icon as specified in the design to achieve the desired look.
![image](https://github.com/okemboi/portfolio/assets/55840999/f43a1eb3-90b5-47dd-9290-820bc936a47f)

By making these style adjustments, your close icon will blend seamlessly with the design, providing a more cohesive and visually appealing user experience. Keep up the good work! 🌟🎨🚀